docs/source/markdown/options/add-host.md

```diff
@@ -2,10 +2,13 @@
 ####>   podman build, create, farm build, pod create, run
 ####> If file is edited, make sure the changes
 ####> are applicable to all of those.
-#### **--add-host**=*host:ip*
+#### **--add-host**=*hostname[,hostname[,…]]*:*ip* | *hostname[,hostname[,…]]*:*host-gateway*
 
-Add a custom host-to-IP mapping (host:ip)
-Multiple hostnames for the same IP can be separated by semicolons.
-
-Add a line to /etc/hosts. The format is hostname:ip or hostname1;hostname2;hostname3:ip if you want to map multiple hostnames to the same ip without duplicating the --add-host parameter. The **--add-host**
-option can be set multiple times. Conflicts with the **--no-hosts** option.
+Adds a custom host-to-IP mapping to the container's `/etc/hosts` file. The
+option takes one or multiple comma-separated hostnames to be mapped to a single
+IPv4 or IPv6 address, separated by a colon. Instead of an IP address the special
+flag *host-gateway* can be given that resolves to an IP address the container
+can use to connect to the host. Podman will add the `host.containers.internal`
+and `host.docker.internal` hostnames using the *host-gateway* address
+automatically, unless the **--no-hosts** option is given. This option can be
+specified multiple times. This option conflicts with the **--no-hosts** option.
```

Comment:
"specified multiple times to add additional mappings to `/etc/hosts` maybe?

Reply:
Yeah, I like that clarification, updated accordingly :+1: